vnet/spec/vnet/endpoints/1.0/translations_spec.rb

```diff
@@ -36,7 +36,6 @@ def app
     let!(:test_interface) { Fabricate(:interface, uuid: 'if-test2') }
 
     accepted_params = {
-      :interface_uuid => "if-test2"
```

Comment:
I missed this one on my first review. This is also still in the webapi, as are `mode` and `passthrough`. They should also be tested or removed from the API.